plugins/org.jkiss.dbeaver.ext.mssql/OSGI-INF/l10n/bundle_tw.properties

```diff
@@ -1,165 +1,173 @@
 # Translated By 2021 Ralic Lo (raliclo@gmail.com)
 # Translated By 2023 Haowei Hsu (hwhsu1231@gmail.com)
 
+<<<<<<< HEAD
+Bundle-Vendor = JKISS
+Bundle-Name = DBeaver SQL Server支持
+
+sqlserver.dialog.connection.header = SQL Server 連線設定
+sybase.dialog.connection.header = Sybase 連線設定
+=======
 sqlserver.dialog.connection.header = SQL Server \u9023\u7DDA\u8A2D\u5B9A
 sybase.dialog.connection.header = Sybase \u9023\u7DDA\u8A2D\u5B9A
+>>>>>>> 70963e96fa47118e5498e2e4b5d7314abc79ab09
```

Comment:
Something is broken here. Probably there were unresolved git conflict, please, fix it.